pages/events/[type].js

```diff
@@ -49,18 +54,20 @@ export default function EventsPage() {
   }
   return (
     <>
-      <Link href="/">Home</Link>
-      <h1>
+      <StyledHeadline>
         Your {type} {type === "wishlist" ? null : "Appointments"}
-      </h1>
-      <button
+      </StyledHeadline>
+
+      <StyledAddButton
```

Comment:
please be explicit about the type of the button :)

Reply:
Done